config_TEST_REGRESSIONS_GAME.ini

```diff
@@ -0,0 +1,36 @@
+[debug]
+
+[main]
+gameFolder = NETHERW ; Main game content folder (after install). Path is relative to .exe 
+cdFolder = CD_Files ; Main game cd content folder (after install). Path is relative to .exe
+
+
+[skips]
+skipintro = true
+
+[sound]
+hqsound = true ; sound is transfered from 11025,8bit,mono to 44000,16bit,stereo
+oggmusic = true ; using AWE32 record of MIDI music, for this function is hqsound auto enabled
+oggmusicFolder = music-ogg ; directory with music, you can rewrite with own music too. Path is relative to .exe
+oggmusicalternative = true ; use original and alternative sound tracks
+fixspeedsound = false ; set true when sounds play double speed
+
+[graphics]
+windowResWidth = 800 ; Window resolution, cannot be lower than gameRes
+windowResHeight = 800 ;
+gameResWidth = 320 ; In Game resolution. If using software render keep this low
+gameResHeight = 320 ;
```

Comment:
AUTO_CHANGE_RES in TEST_REGRESSIONS_GAME is not defined, so it will not switch to a higher resolution. So maybe it would work without the settings in config.ini. But I haven't tried it :)